### PR TITLE
Feature/symbol 35

### PR DESCRIPTION
**Pull request by linababenko**
Modal window for transfer NFT token.

### CODE REVIEW

File: src/components/MaxFeeAndSubmitToken/MaxFeeAndSubmitToken.vue [hunk 1]
```diff
@@ -0,0 +1,114 @@
+<template>
```
Comment (boivlad) [inline src/components/MaxFeeAndSubmitToken/MaxFeeAndSubmitToken.vue:1]: Why did you create a copy of the MaxFreeAndSubmit.vue component? Do you have any differences?

File: src/components/MaxFeeAndSubmitToken/MaxFeeAndSubmitToken.less [hunk 2]
```diff
@@ -0,0 +1,27 @@
+.warning-fee-low {
```
Comment (boivlad) [inline src/components/MaxFeeAndSubmitToken/MaxFeeAndSubmitToken.less:1]: Why did you create a copy of the MaxFreeAndSubmit.less component? Do you have any differences?

File: src/components/MaxFeeAndSubmitToken/MaxFeeAndSubmitToken.vue (hunk 1 above)
Comment (linababenko) [reply src/components/MaxFeeAndSubmitToken/MaxFeeAndSubmitToken.vue:1]: to separate the button "Send" and give it different styles

File: src/components/MaxFeeAndSubmitToken/MaxFeeAndSubmitToken.less (hunk 2 above)
Comment (linababenko) [reply src/components/MaxFeeAndSubmitToken/MaxFeeAndSubmitToken.less:1]: to separate the button "Send" and give it different styles

File: src/views/forms/FormTransferToken/FormTransferToken.vue [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+                                v-model="formItems.signerAddress"
+                                :signers="signers"
+                                @input="onChangeSigner"
+                            />
+
+                            <!-- Transfer recipient input field -->
+                            <RecipientInput v-model="formItems.recipientRaw" style="margin-bottom: 0.5rem;" @input="onChangeRecipient" />
+
+                            <!-- Transfer message input field -->
+                            <MessageInput v-model="formItems.messagePlain" style="margin-bottom: 0.5rem;" @input="onChangeMessage" />
```
Comment (boivlad) [inline src/views/forms/FormTransferToken/FormTransferToken.vue]: It seems to me that these styles are unnecessary for display.

File: src/views/forms/FormTransferToken/FormTransferToken.vue [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+                            <!-- Transaction signer selector -->
+                            <SignerSelector
+                                v-if="!hideSigner"
+                                v-model="formItems.signerAddress"
+                                :signers="signers"
+                                @input="onChangeSigner"
+                            />
+
+                            <!-- Transfer recipient input field -->
+                            <RecipientInput v-model="formItems.recipientRaw" style="margin-bottom: 0.5rem;" @input="onChangeRecipient" />
```
Comment (boivlad) [inline src/views/forms/FormTransferToken/FormTransferToken.vue]: It seems to me that these styles are unnecessary for display.

File: src/views/forms/FormTransferToken/FormTransferToken.vue [hunk 5]
```diff
@@ -0,0 +1,133 @@
+<template>
+    <div class="FormTransferToken">
+        {{ mosaicId }}
```
Comment (boivlad) [inline src/views/forms/FormTransferToken/FormTransferToken.vue]: This is unnecessary value for display

File: src/views/forms/FormTransferToken/FormTransferToken.vue [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+                            </div>
+
+                            <!-- Transaction URI display-->
+                            <FormRow v-if="transactions && transactions.length > 0" class="transaction-uri-display-row">
+                                <template v-slot:inputs>
+                                    <TransactionUriDisplay :transaction="transactions[0]" />
+                                </template>
+                            </FormRow>
+                        </div>
+                        <div class="token-preview">
```
Comment (boivlad) [inline src/views/forms/FormTransferToken/FormTransferToken.vue:65]: Token is very high for modal

**Review by boivlad — CHANGES_REQUESTED** [review]

File: src/views/forms/FormTransferToken/FormTransferToken.vue (hunk 4 above)
Comment (linababenko) [reply src/views/forms/FormTransferToken/FormTransferToken.vue]: Why?

File: src/views/forms/FormTransferToken/FormTransferToken.vue (hunk 3 above)
Comment (linababenko) [reply src/views/forms/FormTransferToken/FormTransferToken.vue]: Why?

Comment (boivlad) [reply src/views/forms/FormTransferToken/FormTransferToken.vue]: The component already has a bottom margin. In this case, there are too many margins between the fields.

File: src/views/forms/FormTransferToken/FormTransferToken.vue (hunk 4 above)
Comment (boivlad) [reply src/views/forms/FormTransferToken/FormTransferToken.vue]: The component already has a bottom margin. In this case, there are too many margins between the fields.

**Review by boivlad — CHANGES_REQUESTED** [review]

**Review by boivlad — APPROVED** [review]

**Pull request merged**